### PR TITLE
Feature/ms 62 fix add product

### PR DESCRIPTION
**Pull request by andrystylist**

### CODE REVIEW

**Review by jlvillegas04 — APPROVED** [review]
LGTM

File: src/components/Admin/Products/Products.jsx [hunk 1]
```diff
@@ -0,0 +1,192 @@
...
+import * as ProductService from '../../../services/ProductService'
+import './Products.scss'
+import ProductTable from './ProductTable'
+
+function Products () {
+  const [isModalOpen, setIsModalOpen] = useState(false)
+  const [formTitle, setFormTitle] = useState('Create Product')
+  const [product, setProduct] = useState() // This is for edition or creation of a product
+  const [search, setSearch] = useState('')
+  const [products, setProducts] = useState([]) // This is a list of products
```
Comment (ht1204) [inline src/components/Admin/Products/Products.jsx:16]: remove not necessary comments

File: src/hooks/useAddProductToCart.js [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+  const handleAddToCart = (product, quantity) => {
+    setProductsList(prevProducts => {
+      const doesProductExist = prevProducts.find((productCart) => productCart.id === product.id)
+
+      if (!doesProductExist) {
+        return [...prevProducts, { ...product, quantity }]
+      }
+
+      // If this reduce is ran the product already exists
```
Comment (ht1204) [inline src/hooks/useAddProductToCart.js:15]: remove this comment

File: src/pages/Checkout/Checkout.jsx [hunk 3]
```diff
@@ -70,21 +79,49 @@ function Checkout () {
...
+
+  useEffect(() => {
+    return () => {
+      if (order) {
+        setProductsList([])
+      }
     }
+  }, [])
+
+  if (productsList.length === 0) {
```
Comment (ht1204) [inline src/pages/Checkout/Checkout.jsx:123]: `if (!productsList.length)`

File: src/pages/Products/Products.jsx [hunk 4]
```diff
@@ -31,16 +42,21 @@ const Products = () => {
...
-        <div className="products-banner">
+        {/* <div className="products-banner">
             <p className="banner-results">Showing 1-9 of 27 results</p>
             <select className="banner-sort">
                 <option value="default">Default sorting</option>
                 <option value="lowest">Price: low to high</option>
                 <option value="highest">Price: high to low</option>
             </select>
-        </div>
+        </div> */}
```
Comment (ht1204) [inline src/pages/Products/Products.jsx:52]: Remove this dead code

File: src/services/ProductService.js [hunk 5]
```diff
@@ -0,0 +1,44 @@
+import { http } from './http'
+
+export const getProductById = (id) => {
+  const controller = new AbortController()
+
+  console.log('--- AC - searchProducts')
```
Comment (ht1204) [inline src/services/ProductService.js:6]: remove this log

File: src/services/ProductService.js [hunk 6]
```diff
@@ -0,0 +1,44 @@
...
+
+export const updateProduct = async (id, product) => {
+  const { data } = await http.patch(`/products/${id}`, product)
+  return data
+}
+
+export const searchProducts = (search) => {
+  const controller = new AbortController()
+
+  console.log('--- AC - searchProducts')
```
Comment (ht1204) [inline src/services/ProductService.js:31]: remove this log

**Review by ht1204 — COMMENTED** [review]
Please, check the comments to correct, thx.

**Review by ht1204 — APPROVED** [review]

**Pull request closed without merging**